### ISSUE TITLE
Default headers doesn't send if I add any custom header

### ISSUE THREAD

**Issue by h1laryz** [swagger-api/swagger-ui#9090]
### Q&A (please complete the following information)
 - OS: ubuntu 22.04.2 LTS
 - Browser: chrome, firefox
 - Version: [e.g. 22]
 - Method of installation: docker compose
 - Swagger-UI version: last docker version
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
When I send any request without my headers they work perfectly, but when I add any header to my request in swagger-ui I am getting error 
... from origin 'http://localhost:11500' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

Request in swagger-ui looks like: 
curl -X 'GET' \
  'http://127.0.0.1:11600/path?date=2023-01-01' \
  -H 'accept: application/json' \
  -H 'x-api-key: <uuid api key here>'

**Comment by robert-hebel-sb** [comment]
We are doing some housekeeping on our issue tracker. This issue has been inactive for over a year and we have not been able to tackle it yet.

Is this still important to you? If so, please leave a comment – we'd love to know if you're still affected. If we don't hear back in 7 days, we'll close this issue to tidy up the tracker.

Thank you for your understanding!

**Comment by robert-hebel-sb** [comment]
No response received after 7 days. Closing this issue to keep the tracker tidy. If you're still affected, please reopen or create a new issue with updated details.
